### ISSUE TITLE
Connect the repo to Reviewable

### ISSUE THREAD

**Issue by userzimmermann** [coala/coala#3400]
Please connect this repo at https://reviewable.io/repositories so that all PRs get linked to reviews. Thanks!

There was this link and I just clicked on it... ;)

**Comment by sils** [comment]
Why? Will try when I have a laptop

**Comment by userzimmermann** [comment]
@sils Just because reviewable.io urged me to do so ;)

**Comment by sils** [comment]
I recall @jayvdb disliked it and I'm strongly against letting a company use
our pull requests as an advertisement platform without good reason

**Comment by jayvdb** [comment]
What I most disliked was it appeared to automatically modify PRs comments that I (and others?) created, to add a button linking to the review on their website.  As a result of their spamming without any opt-in or opt-out that I could see, I didnt get to fully evaluate their product, beyond it being quite a heavy JS app.

**Comment by Mixih** [comment]
Yeah, the reviewable interface seemed quite slow and the way it auto-modified every pull seemed almost spammy. Are there any significant benefits it could provide for our review style?

**Comment by Mariatta** [comment]
I don't see any huge benefit or advantage for doing this.
Just because 'somebrand is telling me to do this' is not good enough reason.
I vote to close this.
